nipype/interfaces/fsl/tests/test_dti.py

```diff
@@ -103,19 +95,20 @@ def setup_tbss():
 def teardown_tbss():
     # Teardown is called after each test to perform cleanup
     os.chdir(test_dir)
-    shutil.rmtree(tbss_dir)
+    rmtree(tbss_dir)
 
 
-@skipif(skip_dti_tests)
+@pytest.mark.xfail(reason="These tests are skipped until we clean up some of this code") 
```

Comment:
This test doesn't run properly. Previously had `@skipif(skip_dti_tests)` ,i.e. `@skipif(True)`. Pytest runs tests that are marked as xfail and if they pass pytest would report `xpassed`.